engine/src/main/java/com/agiletec/aps/system/common/entity/ApsEntityManager.java

```diff
@@ -91,28 +96,32 @@ public abstract class ApsEntityManager extends AbstractService
 
     private IApsEntityDOM entityDom;
 
-    private Map<String, AttributeRole> attributeRoles = null;
-
     private Map<String, String> attributeDisablingCodes = null;
 
     private String attributeRolesFileName;
 
     private String attributeDisablingCodesFileName;
 
     private IEntityManagerCacheWrapper cacheWrapper;
-
+    
     @Override
     public void init() throws Exception {
         this.entityDom.setRootElementName(this.getXmlAttributeRootElementName());
         this.getCacheWrapper().initCache(super.getName());
-        logger.debug("{} : inizializated", this.getName());
+        this.refreshAttributeRoles();
```

Comment:
a refresh operation inside the `init`? ... and we have `refresh` method ... something strange ...

Reply:
done